### ISSUE TITLE
Proposal ref(fleet): use multiple etcd servers

### ISSUE THREAD

**Issue by aledbf** [deis/deis#3848]
With the same conditions mentioned in #3844 I see this behavior in fleet:

```
...
Jun 10 18:05:16 nodo-4 fleetd[692]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Jun 10 18:05:16 nodo-4 fleetd[692]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Jun 10 18:05:16 nodo-4 fleetd[692]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Jun 10 18:05:16 nodo-4 fleetd[692]: WARN engine.go:115: Engine completed reconciliation in 4.696940778s
...
```
(approximately every 10 minutes)
and that trigger a re-scheduling of the running units in the node.

and then
```
....
Jun 10 18:31:45 nodo-4 systemd[1]: Starting Network Service...
Jun 10 18:31:46 nodo-4 systemd-networkd[27471]: vethc2e8385     : could not find udev device: No such device
Jun 10 18:31:46 nodo-4 systemd-networkd[27471]: could not add new link: No such device
Jun 10 18:31:46 nodo-4 systemd-networkd[27471]: Could not enumerate links: Device or resource busy
Jun 10 18:31:46 nodo-4 systemd[1]: systemd-networkd.service: main process exited, code=exited, status=1/FAILURE
Jun 10 18:31:46 nodo-4 systemd[1]: Failed to start Network Service.
Jun 10 18:31:46 nodo-4 systemd[1]: Unit systemd-networkd.service entered failed state.
Jun 10 18:31:46 nodo-4 systemd[1]: systemd-networkd.service failed.
Jun 10 18:31:46 nodo-4 systemd[1]: systemd-networkd.service has no holdoff time, scheduling restart.
Jun 10 18:31:46 nodo-4 systemd[1]: Starting Network Service...
Jun 10 18:31:46 nodo-4 systemd-networkd[27474]: Enumeration completed
Jun 10 18:31:46 nodo-4 systemd[1]: Started Network Service.
Jun 10 18:31:46 nodo-4 systemd-networkd[27474]: veth78af1d4     : link configured
Jun 10 18:31:46 nodo-4 systemd-networkd[27474]: veth39a6795     : link configured
...
```

Adding more than one etcd server in the fleet unit using `FLEET_ETCD_SERVERS` removes this strange behavior

**Comment by bacongobbler** [comment]
I think this is resolved with #3679, if I'm not mistaken. :)

CC @krancour

**Comment by aledbf** [comment]
@bacongobbler  @krancour I'm closing this because it makes no sense. 
Passing multiple urls makes no difference because always use the first one and unless the order of the servers is different it will use the same etcd server for all the traffic.
